notebooks/D1-entitlement-demographics.ipynb

```diff
@@ -15,73 +15,148 @@
    "source": [
     "import intake\n",
     "import IPython.display\n",
+    "import ipywidgets\n",
     "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
     "import pandas\n",
+    "import pcts_census_utils\n",
     "\n",
     "cat = intake.open_catalog(\"../catalogs/*.yml\")"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 2,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Load PCTS extract\n",
-    "pcts = cat.pcts2.read().reset_index(drop=True)"
+    "## Load data"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ACS data for income, race, commute, tenure\n",
-    "census = cat.census_analysis_table.read()"
+    "# Load PCTS and subset\n",
+    "parent_cases = cat.pcts_parents.read()\n",
```

Comment:
One thing I don't understand: what is the reason for a separate `pcts_parents` parquet? Isn't it pretty easy to filter out child cases directly? I kind of liked the regex approach since it was closer to the original dataset, and was pretty fast (much faster than using the `PCTSParser` wrapper).

Reply:
Then i think the change needs to be made in the `pcts_census_utils`, instead of calling the `PCTSParser` wrapper, it does a regex there?

Reply:
Yeah, that's right. Basically, I'm wondering if we can get rid of `pcts_parents` all together, and just make it less painful to parse PCTS by eschewing the `PCTSParser` class.

Reply:
The parent cases doesn't have to be separate, as long as the `subset_pcts` function includes matching it against parent cases to filter out child cases. It can be all done in the same function, but, I prefer it to be separate in case we need all PCTS cases for something, we can still subset it, without subsetting + dropping child cases in one fell swoop. 

It can be done with 2 functions, where the 1 high level function to used in most of our notebooks to do subsetting and dropping child cases.

Reply:
Do you want the `PCTSParser` removed completely? Or leave it, because it still parses out overlays, height districts, development limits, and not just the prefixes and suffixes?

Reply:
Oh, right, but we pivot the parent cases so that we make sure to pick up suffixes in any child cases, right? So any util would need to do that.

Reply:
I think `PCTSParser` is okay to keep, it's just very slow, and using the underlying regex was useful for processing the whole dataset.